data/json/recipes/electronics/parts.json

```diff
@@ -119,6 +119,21 @@
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "components": [ [ [ "scrap", 6 ] ], [ [ "cable", 4 ] ], [ [ "plut_cell", 20 ] ] ]
   },
+  {
+    "result": "flux_comp_gen",
+    "type": "recipe",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "electronics",
+    "difficulty": 5,
+    "skills_required": [ "fabrication", 4 ],
+    "time": "60 m",
+    "book_learn": [ [ "textbook_anarch", 5 ], [ "advanced_electronics", 5 ] ],
+    "using": [ [ "forging_standard", 3 ], [ "soldering_standard", 10 ] ],
+    "qualities": [ { "id": "SCREW", "level": 1 }, { "id": "SMOOTH", "level": 1 } ],
+    "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ] ],
+    "components": [ [ [ "cu_pipe", 2 ] ], [ [ "cable", 241 ] ], [ [ "composite_b", 450 ] ], [ [ "concrete", 1 ] ], [ [ "water", 6 ] ] ]
```

Comment:
Could use clean water in place of water as well. Also probably at least a CONTAIN 1 quality if you're actually mixing concrete by hand.